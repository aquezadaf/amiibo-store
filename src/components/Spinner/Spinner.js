import React from 'react';
import { useTranslation } from 'react-i18next';
import spinner from './img/spinner.svg'
import styles from './Spinner.module.css'

export default function Spinner() {
  const { t } = useTranslation();
  return (
    <div className={styles.loader}>
      <img src={spinner} alt="spinner" />
      <div className={styles.text}>{t('loading')}</div>
    </div>
  );
}
