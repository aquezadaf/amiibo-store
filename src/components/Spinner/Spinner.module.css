.loader {
    margin: 0 0 2em;
    height: 100px;
    width: 20%;
    text-align: center;
    padding: 1em;
    margin: 0 auto 1em;
    display: inline-block;
    vertical-align: top;
}

.text {
    font-size: 15px;
    font-weight: bold;
}
