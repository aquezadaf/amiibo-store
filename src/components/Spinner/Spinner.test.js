import React from 'react'
import { shallow } from 'enzyme'
import Spinner from './Spinner'

it('renders spinner', () => {
    const wrapper = shallow(<Spinner />)
    expect(wrapper.html()).toMatchSnapshot()
})
