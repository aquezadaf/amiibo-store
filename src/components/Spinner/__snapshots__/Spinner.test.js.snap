// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner 1`] = `"<div class=\\"loader\\"><img src=\\"spinner.svg\\" alt=\\"spinner\\"/><div class=\\"text\\">loading</div></div>"`;
